src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const populations = document.querySelectorAll('.population');
+const totalNumber = document.querySelector('.total-population');
+const averageNumber = document.querySelector('.average-population');
+
+const populationNumbers = [...populations].map(element =>
+  Number(element.innerText.split(',').join('')));
+
+const totalPopulation = populationNumbers.reduce(
+  (sum, element) => sum + element, 0);
```

Comment:
You don't have to iterate your array twice, you can just do all your logic in reduce

Reply:
@kirill-vypirovskyi Good point.
@lilia-obushenko Implement this.